### ISSUE TITLE
False positive malformed input for `#![feature()]` 

### ISSUE THREAD

**Issue by neonaot** [intellij-rust/intellij-rust#10419]
## Environment

* **IntelliJ Rust plugin version:** 0.4.194.5366-231-beta
* **Rust toolchain version:** 1.71.0-nightly (9ecda8de8 2023-04-30) x86_64-apple-darwin
* **IDE name and version:** CLion 2023.1.2 (CL-231.8770.66)
* **Operating system:** macOS 12.5.1
* **Macro expansion:** enabled
* **Additional experimental features:** org.rust.macros.proc.attr, org.rust.cargo.features.settings.gutter, org.rust.ide.intentions.macros.function-like, org.rust.macros.proc, org.rust.ssr

## Problem description
That is useless code, but the compiler doesn't think it's an error.
<img width="698" alt="image" src="https://user-images.githubusercontent.com/49211026/235631994-2fcdffd0-bd59-4631-81e3-137b47faafe2.png">

Related to https://github.com/intellij-rust/intellij-rust/pull/10359

## Steps to reproduce
```rust
#![feature()]
fn main() {}
```
